scripts/jenkins/cloud/ansible/roles/list_packages/tasks/diff-packages.yml

```diff
@@ -0,0 +1,55 @@
+#
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+
+- name: Diff list of installed packages in plaintext
+  shell: |
+    diff -U 0 /tmp/rpms/rpms_{{ item }}_{{ state1 }}.txt \
```

Comment:
The diff format is really not very friendly to look at, but it will have to do for now. In the future, we'll have to replace this with a yaml format, same as the other list of packages.

Reply:
Yes I agree. Victor tried to use python to do  yaml diff but it had broken module so I removed it for now.